code-challenges.test.js

```diff
@@ -85,22 +87,22 @@ const object3 = { number: -7 }
 
 
 
-// describe("isDivisible",() => {
-//     it("decides if the number inside it is evenly divisible by three or not", () => {
-//         expect(isDivisible(object1.number)).toEqual ("15 is divisible by three")
-//         expect(isDivisible(object2.number)).toEqual ("0 is divisible by three")
-//         expect(isDivisible(object3.number)).toEqual ("-7 is not divisible by three")
-//     })
-// })
+describe("isDivisible",() => {
+    it("decides if the number inside it is evenly divisible by three or not", () => {
+        expect(isDivisible(object1.number)).toEqual ("15 is divisible by three")
+        expect(isDivisible(object2.number)).toEqual ("0 is divisible by three")
+        expect(isDivisible(object3.number)).toEqual ("-7 is not divisible by three")
+    })
+})
 
 // // b) Create the function that makes the test pass.
 
-// const isDivisible = (object) => {
-//     if (object% 3 === 0 )  {
-//         return `${object} is divisible by three`
-//     }
-//     else {return `${object} is not divisible by three`}
-// }
+const isDivisible = (object) => {
+    if (object% 3 === 0 )  {
+        return `${object} is divisible by three`
+    }
+    else {return `${object} is not divisible by three`}
+}
```

Comment:
With the parameter being an object, you should be accessing the key value pairs within your function instead of in the test. Also, make sure you are explaining how you access the values and what your conditions are in your pseudo code.